src/main/java/task/Event.java

```diff
@@ -0,0 +1,29 @@
+package task;
+
+import java.time.LocalDate;
+
+import utility.DateParser;
+
+public class Event extends Task {
+    private final String eventTime;
+    private LocalDate date = null;
+
+    public Event(String name, String eventTime) {
+        super(name);
+        this.eventTime = eventTime;
+        try {
+            date = LocalDate.parse(eventTime);
+        } catch (Exception e) {
```

Comment:
Consider catching a more specific exception such as DateTimeParseException that is thrown by the parse method when the text cannot be parsed. Hovering over methods can provide some information on specific exceptions thrown by library methods.